package.json

```diff
@@ -11,7 +11,7 @@
     "fmt": "prettier --write '**/*.{js,jsx,ts,tsx,json}'",
     "fmt:check": "prettier --check '**/*.{js,jsx,ts,tsx,json}'",
     "build": "npm run fmt && node ./module/include.js",
-    "dev": "~/.cargo/bin/bos-loader devgovgigs.near --path src",
+    "dev": "~/.cargo/bin/bos-loader devhub.efiz.testnet --path src -r replacements.dev.json",
```

Comment:
Is there a way to make this developer independent?

Reply:
This was my mistake, I didn't mean to include it; this was remnant from testing something else.

Reply:
Although I do agree it should be easy to change the account or contract id without modifying code in multiple places as discovery team does.

I replaced this line with a dev script which allows you to provide your own --account (-a), --contract (-c), or --network (-n). Otherwise it defaults to the devhub mainnet, testnet, or testnet-dev account